src/Components/Inputs/Input.tsx

```diff
@@ -173,6 +182,19 @@ export const Input = forwardRef<InputRef, InputProps>(
 
     const iconElement = icon && <Icon glyph={icon} className="cf-input-icon" />
 
+    const clearClasses = classnames('cf-input-clear-btn', {
+      large: size === ComponentSize.Large,
```

Comment:
need to account for extra small, and medium as well

Reply:
done.